engines/catalog/spec/services/catalog/product_import/products_reset_strategy_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 module Catalog
   module ProductImport
-    describe ProductsResetStrategy do
+    RSpec.describe ProductsResetStrategy do
```

Comment:
I'd seen this format online so many times but never bothered to find out why people use it. Now I know!
I would love to see a neater solution, but don't know enough to suggest one ;)

Reply:
In this example, we put the spec in a module which is bad practice. But otherwise The `RSpec.` is only at the top of the file and I don't mind that. It makes it very clear, less magic and line length is usually not a problem at the top level of the spec. Oh, and spec generators add it anyway. I used to manually remove it but now I will have less work after generating a model with a spec.